EIPS/eip-6160.md

```diff
@@ -0,0 +1,80 @@
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-01-31
+requires: 20, 5679, 5982
+---
+
+## Abstract
+
+We propose new [`ERC20`](./eip-20.md), [`ERC721`](./eip-721.md) and [`ERC1155`](./eip-1155.md) extension interfaces which will allow provably secure, trustless bridge contracts to mint & burn tokens, enabling a new class of `ERC` tokens that can be native to multiple chains.
+
+## Motivation
+
+[EIP-4844](./eip-4844.md) will set the stage for the Cambrian explosion of L2s on Ethereum. With multiple L2s living on the ethereum blockchain, It therefore becomes necessary to standardize an interface that will allow for native `ERC` tokens to flow across these L2s, removing the need for wrapped representations of these assets.
+
+Optimisim and Arbitrum, optimistic L2s which are already live on Ethereum today currently have their own custom token interfaces¹ that allow their bridge contracts to mint & burn tokens on L1 & L2. In order to remove the cognitive load on token issuers & developers from extending multiple interface for all the various L2s that will live on Ethereum. It becomes necessary to define a standardized API that all token issuers should implement in order to provide these trustless bridge contracts with the interface needed to transport tokens across these different chains.
+
+## Specification
+
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+Smart contracts implementing the EIP-6160 standard MUST implement all of the functions in the `EIP-6160` interface.
+
+Smart contracts implementing the EIP-6160 standard MUST implement the `EIP-165 supportsInterface` function and MUST return the constant value `true` if *`0x01cbdea7`* is passed through the `interfaceID` argument.
+
+1. Any contract complying with [EIP-20](./eip-20.md) when extended with this EIP, **MUST** implement the following interface:
+
+```solidity
+// The EIP-165 identifier of this interface is 0xbbb8b47e
+interface IERC6160Ext20 is IERC5679Ext20, IERC_ACL_CORE {}
+```
+
+2. Any contract complying with [EIP-721](./eip-721.md) when extended with this EIP, **MUST** implement the following interface:
+
+```solidity
+// The EIP-165 identifier of this interface is 0xa75a5a72
+*interface IERC6160Ext721 is IERC5679Ext721, IERC_ACL_CORE {}
+```
+
+3. Any contract complying with [EIP-1155](./eip-1155.md) when extended with this EIP, **MUST** implement the following interface:
+
+```solidity
+// The EIP-165 identifier of this interface is 0x9f77104c
+interface IERC6160Ext1155 is IERC5679Ext1155, IERC_ACL_CORE {}
+```
+
+4. It is RECOMMENDED for compliant contracts to implement the optional extension `IERC_ACL_GENERAL`.
+5. Compliant contracts MAY implement the optional extension `IERC_ACL_METADATA`.
+
+## Rationale
+
+1. We have chosen the [EIP-5679](./eip-5679.md) standard to provide the mint & burn interface.
```

Comment:
The rationale section should explain _why_ a choice was made, not just state which decisions were considered.